scripts/Muon/GUI/FrequencyDomainAnalysis/FFT/fft_presenter.py

```diff
@@ -31,21 +41,25 @@ def deactivate(self):
         self.view.deactivateButton()
 
     def getWorkspaceNames(self):
-        final_options=self.load.getWorkspaceNames()
+        final_options = self.load.getWorkspaceNames()
         self.view.addItems(final_options)
 
-    #functions
+    # functions
     def phaseCheck(self):
```

Comment:
This function is doing more than just checking isn't it, by calling `setPhaseBox`. So is there a more descriptive name.

Reply:
if its not there then we are not allowing users to decide not to produce one